Utils/api.js

```diff
@@ -60,5 +60,16 @@ export function deleteComment(comment_id) {
     });
 };
 
+export function getTopics() {
+    return axios
+    .get(`https://ellsbees-nc-news.onrender.com/api/topics`).then(({ data }) => {
+    return data;
+    });
+};
 
-
+export function getArticlesByTopic(slug) {
```

Comment:
Rather than creating a new API call here, you could use axios params on your existing getArticles function. This would allow you to only pass in a topic if it's been selected, so if it was undefined it would still fetch all the articles. It would also mean you can reuse your ArticleList component to display the articles by topic 👍